Extensions/CMake/SlicerExtensionPackageAndUploadTarget.cmake

```diff
@@ -19,15 +20,19 @@
 #  Slicer_REVISION
 #  EXTENSION_NAME
 #  EXTENSION_CATEGORY
-#  EXTENSION_ICONURL
-#  EXTENSION_CONTRIBUTORS
-#  EXTENSION_DESCRIPTION
-#  EXTENSION_HOMEPAGE
-#  EXTENSION_SCREENSHOTURLS
 #  EXTENSION_ENABLED
 #  EXTENSION_OPERATING_SYSTEM
 #  EXTENSION_ARCHITECTURE
 #
+# The following variables are internally set by extracting corresponding values
+# from the locally generated "<extension_name>.s4ext" file:
```

Comment:
.s4ext or .json?

Reply:
A `.s4ext` is still generated in the extension build tree and used, it should now be considered an implementation details for now. This will be addressed later.

Reply:
Maybe clarify this in the commit comment and maybe add a comment in the code that the .s4ext file is for internal use only and may be changed to different file format later.